ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/commands/tracker/CommandTracker.scala

```diff
@@ -279,33 +292,70 @@ private[commands] class CommandTracker[Context](
       }
 
       private def getOutputForCompletion(completion: Completion) = {
-        val (commandId, errorText) = {
+        val (potentialCommandKey, errorText) = {
```

Comment:
```suggestion
        val (potentialCommandKey, tracingText) = {
```

Reply:
A cleaner alternative (i.e. that doesn't mix logging and regular logic) would perhaps to just build the logged string based on the completion status.

Reply:
Renaming to `completionDescription`, as `tracing` may be confused with telemetry.